keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -1249,6 +1537,10 @@ well as the [existing list] of feature gates.
       - kube-apiserver
       - kube-controller-manager
       - kube-scheduler
+      - kubelet
+  - Feature gate name: KubeletPodFailureConditions
```

Comment:
Why adding a different flag to kubelet, which would be disconnected from `PodDisruptionConditions` such that we could get only half of the future working?

Reply:
The motivation was that not all conditions added by Kubelet would be disruptions (but also`ResourceExhausted`). 

If you think this is not a problem, then I prefer to simplify it and stay only with `PodDisruptionConditions`. 

Alternatively, could we maybe rename the `PodDisruptionConditions` gate to `PodFailureConditions`?

@soltysh @alculquicondor ?

Reply:
Renaming is not backwards compatible. I think we can just the same feature gate. The good thing is that users can disable the gate in kubelet but not in the control plane :)

Reply:
Removed the new flag

Reply:
> Renaming is not backwards compatible. I think we can just the same feature gate. The good thing is that users can disable the gate in kubelet but not in the control plane :)
Correct, you still have fine grain control over where it's enabled, but having the same name makes it explicit and easier to understand. Thx!